File: lw7/2/Vasilyev_Andrey/DataWriter.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace lab7
{
    public class DataWriter
    {

        public static void WriteToFile(List<int> data, string outputFileName = "output.txt")
        {
            // Write the string to a file.
            System.IO.StreamWriter file = new System.IO.StreamWriter(outputFileName);
            for (int i = 0; i < data.Count; ++i)
            {
                file.Write(data[i]);
                file.Write(" ");
            }
            file.Close();
        }
    }
}

File: lw6/2_Vasilyev_Andrey/DataWriter.cpp
#include "DataWriter.h"
#include "stdafx.h"


void DataWriter::WriteToFile(const std::vector<int> & data, const std::string & outputFileName)
{
	std::ofstream outputFile(outputFileName);
	for (auto el : data)
	{
		outputFile << el << " ";
	}
	outputFile.flush();
}